pkg/filesystem/local_directory_windows.go

```diff
@@ -584,44 +585,50 @@ func (d *localDirectory) Remove(name path.Component) error {
 	return err
 }
 
-// On NTFS mount point is a reparse point, no need to unmount.
 func (d *localDirectory) RemoveAllChildren() error {
+	return d.RemoveAllChildren(nil)
+}
+
+// On NTFS mount point is a reparse point, no need to unmount.
+func (d *localDirectory) RemoveAllChildren(dPath *path.Trace) error {
 	defer runtime.KeepAlive(d)
 
 	names, err := readdirnames(d.handle)
 	if err != nil {
-		return err
+		return util.StatusWrapf(err, "Failed to read contents of directory %#v", dPath.String())
 	}
 	for _, name := range names {
 		component := path.MustNewComponent(name)
 		fileType, err := d.lstat(component)
 		if err != nil {
-			return err
+			return util.StatusWrapf(err, "Failed to stat component %#v in %#v", name, dPath.String())
```

Comment:
Could you please use `childPath` in all of these error messages?

Reply:
Done, in subsequent work.